test/electron/index.js

```diff
@@ -0,0 +1,13 @@
+var assert = require('assert');
+var evaluate = require('../../lib/util/versioning').evaluate;
+var find = require('../../').find;
+var path = require('path')
+
+describe('electron test', function() {
+    it('just pass', function() {
+        assert.equal(1,1);
+        var opts = evaluate(require('../app1/package.json'));
+        console.log(opts)
+        console.log(find(path.resolve(path.join(__dirname,'../app1/package.json'))))
```

Comment:
This was a leftover from an attempt to actually perform an integration test with electron, but there are some costs to adding those in like adding additional CI prebuilt downloads.  I decided to just go with the unit tests in `test/versioning.test.js` which seem to cover the issue at play here.  Removing this file for now.